packages/ui/src/components/PriceWidget/index.tsx

```diff
@@ -3,6 +3,7 @@ type PriceTypes = {
   label: string;
   currency: string;
   id: string;
+  uuid?: string;
 };
```

Comment:
Moeten de type niet importeren uit `types.d.ts`?

Reply:
Ik heb geen `types.d.ts` in de `@frameless/ui`. Die heb ik in de `@frameless/strapi-tiptap-editor`
https://github.com/frameless/strapi/pull/622/files#diff-963810e079f624fe2b884caf99d34d699d5204a3a6b6ac4ea71e8e8e78adc3cf